openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py

```diff
@@ -251,16 +251,21 @@ def create_bulk_transfer(self):
     def get_auth_token(self):
         try:
             headers = {
-                'Platform-TenantId': os.environ.get("tenantName"),
-                'Authorization': 'Basic Y2xpZW50Og==',
-                'Content-Type': 'text/plain',
+                "Platform-TenantId": os.environ.get("tenantName"),
+                "Authorization": "Basic Y2xpZW50Og==",
```

Comment:
this field should not be added to code, as it is a credential, it should be configured outside code

Reply:
Its taking from environment variable .

Reply:
` "Authorization": "Basic Y2xpZW50Og==",`   This line